racket.html.markdown

```diff
@@ -164,10 +165,16 @@ my-pet ; => #<dog>
 (cons 1 (cons 2 (cons 3 null))) ; => '(1 2 3)
 ;; `list' is a convenience variadic constructor for lists
 (list 1 2 3) ; => '(1 2 3)
-;; and a quote can also be used for a literal list value
+;; a quote can also be used for a literal list value
 '(1 2 3) ; => '(1 2 3)
+;; a quasiquote with commas can be used to evaluate functions
+`(1 ,(+ 1 1) 3) ; => '(1 2 3)
```

Comment:
Okay, you should probably mention that the ` backtick mark is acts as a quasiquote in Racket then.